index.js

```diff
@@ -0,0 +1,66 @@
+function shuffle_array(array) {
+    for (let index = array.length - 1; index > 0; index--) {
+        let random_index = Math.floor(Math.random() * (index + 1));
+        let temp = array[index];
+        array[index] = array[random_index];
+        array[random_index] = temp;
+    }
+    return array
+}
+
+function get_new_question() {
+    if (remaining_movies.length < 1){
+        let user_feedback_element = document.querySelector('#user_feedback');
+        user_feedback_element.innerHTML = '<h3>You have run out of questions</h3>';
+    } else {
+        let current_movie = remaining_movies.pop();
+        let quote_element = document.querySelector('#quote');
+        quote_element.textContent = current_movie.quote;
+
+        let answer_element = document.querySelector('#answer');
+        answer_element.style.display="none";
+        answer_element.innerHTML = '<h2>' + current_movie.title + '</h2>';
+
+        let correct_incorrect_elem = document.querySelector('#correct_incorrect');
+        correct_incorrect_elem.style.display="none";
+    }
+}
+
+let remaining_movies = {}
+document.getElementById('submit_button').addEventListener("click", function () {
+    let guess = document.getElementById('guess')
+    let player_guess = guess.value
+    console.log(player_guess)
+    let answer = document.getElementById('answer').textContent
+    console.log(answer)
+    let correct_response = document.getElementById('correct_response')
+    let incorrect_response = document.getElementById('incorrect_response')
+    if(player_guess.toLowerCase() === answer.toLowerCase()) {
+        correct_response.style.display="block";
+        incorrect_response.style.display="none";
+        guess.value = ''
+        document.getElementById('next_question_button').disabled = false
+        console.log('correct')
+    } else {
+        correct_response.style.display="none";
+        incorrect_response.style.display="block";
+        console.log('incorrect')
```

Comment:
Remove any console.log()s, they should never actually go into the main branch

Reply:
Done